packages/react/src/components/button/button.tsx

```diff
@@ -135,7 +135,7 @@ const Button = React.forwardRef((props, forwardedRef) => {
 
   return (
     <StyledButton
-      {...mergeProps(buttonProps, focusProps, hoverProps, other)}
+      {...mergeProps(buttonProps, focusProps, hoverProps)}
```

Comment:
Paired with @p44dyoachim on this - `other` should be removed from props as the hook `useButton` returns all mapped html attributes correctly in `buttonProps`

Reply:
@dmorton-p44 @p44dyoachim This is not safe to remove other prop from mergeProps, this will cause issue in working of tooltip, as far as I understand this other includes some handlers which are being utilized when the button is wrapped with Tooltip i.e things like tooltip trigger state.

I tried locally running storybook and checking Tooltip over button without the other prop, it seems to not trigger the tooltip and as soon as I add other prop back it seems to work fine.

Reply:
In this case - we should be isolating what those props are from tooltip and passing them explicity, and avoid passing the ambiguous `other` to this mergeProps function

Reply:
It looks like removing `other` props cause this. 
Can we revert these changes? 

<img width="553" alt="image" src="https://github.com/project44/manifest/assets/109169910/4854b26e-cd31-49f1-85f3-1361903d7f7d">

Reply:
The toggle **before** manifest update:
<img width="408" alt="image" src="https://github.com/project44/manifest/assets/109169910/3414ca49-8c3f-4e61-b975-f1edf499bf5a">